### ISSUE TITLE
Bridging from mastodon to bluesky stopped again

### ISSUE THREAD

**Issue by roelgrif** [snarfed/bridgy-fed#1563]
Bridging from mastodon to bluesky seemed to work again briefly, but now it stopped again.

**Comment by snarfed** [comment]
Hmm! Which account is this?

**Comment by AtiusAmy** [comment]
On *Key currently it is not working too hmmm

I replied: 
https://social.atiusamy.com/notes/a13zfrq5pccp02wd

The original post doesn't have it:
https://bsky.app/profile/did:plc:m3yw7j3qfna7k3gj5k4poohj/post/3lbvscjxlds2s

Here's another case
I posted this a while back:
https://bsky.app/profile/amyiscoolz.atiusamy.com/post/3lbv7we6vgag2

I got a reply (this account is also bridged), but this reply doesn't show on bsky:
https://sakurajima.social/notes/a133w6dbgm

**Comment by Sterophonick** [comment]
> > Mine just never worked at all thus far, my instance, https://labyrinth.zone/, recently unblocked the brid.gy domain, yet things still are not working
> > 
> > BSky bridge: https://bsky.app/profile/Sterophonick.labyrinth.zone.ap.brid.gy Main account: https://labyrinth.zone/Sterophonick
> > 
> > Worth mentioning that my instance is Akkoma-based, but I'm not sure if that should be an issue since it's still ActivityPub.
> 
> Akkoma is currently broken :( #374  AFAIK *key, mastodon-based, and GTS are the ones known working (maybe someone can add more to this list (also wanna test out iceshrimp.NET as well and see if it works))
> 
> (Okay GoToSocial is a weird one, it works for my boyfriend https://bsky.app/profile/winsdominoes.fedi.winscloud.net.ap.brid.gy but apparently your mileage may vary  #1033)
> 

Is that on the end of Akkoma or Brid.gy? If it's Akkoma that unfortunately means that unblocking the domain on our end was all for nothing since Akkoma is seldom updated

**Comment by snarfed** [comment]
Looks like on Akkoma's end: https://github.com/snarfed/bridgy-fed/issues/374#issuecomment-2302801799

**Comment by roelgrif** [comment]
Account @roelgrif.mstdn.social.ap.brid.gy still not bridged from https://mstdn.social/@roelgrif

**Comment by snarfed** [comment]
@roelgrif have you looked at https://fed.brid.gy/docs#troubleshooting ?

**Comment by roelgrif** [comment]
I did. Nothing wrong so to see. But somehow the bridging started again shortly after my last comment here. I seem to experience long delays for bridging, but in the end they arrive.

**Comment by snarfed** [comment]
Duplicate of #374
